content/en/serverless/installation/dotnet.md

```diff
@@ -13,69 +13,188 @@ further_reading:
   text: 'Submitting Custom Metrics from Serverless Applications'
 ---
 
-## Prerequisites
+To instrument your .NET serverless application, you must use **either** the [Datadog Lambda Extension][1] (beta) or the [Datadog Forwarder Lambda function][2].
 
-To instrument your .NET serverless application, you must install **either** the [Datadog Forwarder Lambda function][1] or (beta) the [Datadog Lambda Extension][2].
+If you use the Datadog Lambda Extension, you can use Datadog's native .NET APM tracer to instrument your function. If you use the Datadog Forwarder Lambda function, you can use Datadog's integration with AWS X-Ray tracing.
```

Comment:
Should we make this simpler by saying if you want to use tracing, you need to use the Extension?

Reply:
I'll leave it to you. However, some customers might want to keep using the Forwarder (e.g. they are using ARM or they don't want to use a beta product) and these customers probably do want to use the X-Ray integration.